README.md

```diff
@@ -25,7 +28,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
 
-<img src="figures/dependencies.png" width="863" height="347" />
+<img src="figures/dependencies.png" width="100%" />
```

Comment:
on github the width of README is fixed but for example on [npm](https://www.npmjs.com/package/fantasy-land) it looks bad because max width of readme area is 727px and if you are on small screan than it's worth.